docs/topologies/star-shaped.md

```diff
@@ -0,0 +1,53 @@
+# Star-Shaped Head Network
+
+:hammer_and_wrench: This document is a work in progress
+
+This document details the behaviour of so-called _Star-shaped Hydra Network_.
+
+## Summary
+
+A _Star-shaped Hydra Network_ or more precisely a Star-shaped Heads Network is comprised of:
+
+* A central _Server_ node providing "Head-as-a-service", with low expected downtime, probably operated by some company or organisation with enough resources to host this service,
```

Comment:
Is there anything in the design forcing the "server" to be **one**? Conceptually, that's what we represent indeed, but in practice I imagine the requirement to be a bit more permissive: what is required it to have _some_ Hydra head that is somewhat stable with low downtime with which clients can interact by forming channels with one of the participant?

Perhaps this is too much complication already and more about an extension of the base start-shape design :thinking:

Reply:
I would say this should be a next step. Having _one_ "Server" makes things simpler for "Clients" because they would know where to look for, that's even a service one could pay for. But yes, in principle, it could be just a way to bridge heads.